ingest-app/src/cmr/ingest/validation/additional_attribute_validation.clj

```diff
@@ -39,8 +39,8 @@
   attributes that are still referenced by existing granules. This function build the search parameters
   for identifying such invalid deletions."
   [aas prev-aas]
-  (let [aa-names (set (map :Name aas))
-        deleted-aas (remove #(aa-names (:Name %)) prev-aas)]
+  (let [aa-names (set (map #(lower-case (:Name %)) aas))
```

Comment:
You have a lot of uses of `#((lower-case (:Name %))` here. It might be cleaner to define a function and reference it so you don' t need all the anonymous functions.

```
(defn- aa->lowercase-name
  [aa]
  (-> aa :Name lower-case))
``